MetroscopeModelingLibrary/Partial/HeatExchangers/WaterFlueGasesMonophasicHX.mo

```diff
@@ -135,6 +138,12 @@ equation
   // The estimation of the Cp outlet is calculated for an outlet temperature based on the nominal temperature rise of the H&MB diagram.
   // For more details about this hypothesis, please refer the Economiser page of the MML documentation.
 
+  // Nominal temperature differences
+  if nominal_DT_default then
+    nominal_cold_side_temperature_rise = hot_side.T_in - cold_side.T_in; // Corresponds to the maximum achievable temperature difference
+    nominal_hot_side_temperature_drop = hot_side.T_in - cold_side.T_in; // Corresponds to the maximum achievable temperature difference
+  end if;
+
```

Comment:
There is a copy paste issue, both in equation and in comment here 😇

Reply:
The maximum DT applies for both the rise and drop. In other words, the maximum temperature that the cold_side can perform is reaching hot_side.T_in, and the the minimum temperature that the hot_side could reach is cold_side.T_in